broadway_api/daos/worker_node.py

```diff
@@ -21,18 +19,19 @@ def __init__(self, app):
 
     def insert(self, obj):
         document = self._to_store(obj)
-        del document[WorkerNodeDao.ID]
         return self._collection.insert_one(document)
 
+    def update(self, obj):
+        document = self._to_store(obj)
+        return self._collection.update_one(
+            {WorkerNodeDao.ID: obj.id}, {"$set": document}
+        )
+
     def find_all(self):
         return list(map(self._from_store, self._collection.find()))
 
-    def find_by_id(self, id_):
-        if not ObjectId.is_valid(id_):
-            return None
-        return self._from_store(
-            self._collection.find_one({WorkerNodeDao.ID: ObjectId(id_)})
-        )
+    def find_by_id(self, id):
```

Comment:
Can we rename `id` to `_id`? It shadows the builtin `id` function

Reply:
ok